.github/workflows/mutalkCI.yml

```diff
@@ -0,0 +1,92 @@
+name: MutalkCI
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+    - 'README.md'
+    - 'resources/**'
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+    - 'README.md'
+    - 'resources/**'
+
+  workflow_dispatch:
+#    inputs:
+#        mode:
+#           description: 'modes: full (run on all project), diff (run on the last commit diff), commit (especify a commit to calculate diff)'
+#          required: true
+#          default: 'diff'
+#        commit:
+#          description: 'If mode == commit, here you enter the Commid id/name/shortId to evaluate the diff'
+#          required: false
+#          default: ''
+
+jobs:
+  eventBased:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'workflow_dispatch'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: hpi-swa/setup-smalltalkCI@v1
+        id: smalltalkci
+        with:
+          smalltalk-image: Pharo64-stable
+
+      - run: smalltalkci -s ${{ steps.smalltalkci.outputs.smalltalk-image }}
+        shell: bash
+        timeout-minutes: 15
+
+      - run: ${{env.SMALLTALK_CI_VM}} ${{env.SMALLTALK_CI_IMAGE}} metacello install "github://matburnx/mutalk:-CI-not-working" BaselineOfMuTalk
```

Comment:
This (and above) is pointing to the fork.

Reply:
It's temporary, as the official doesn't have any working implementation, it will be changed soon when the pull request will be approved.